src/Stripe.net/Services/_base/Service.cs

```diff
@@ -252,8 +252,23 @@ protected TEntityReturned CreateEntity(BaseOptions options, RequestOptions reque
                 options,
                 requestOptions);
 
+            options = options ?? new ListOptions();
+            bool iterateBackward = false;
+
+            // Backward iterating activates if we have an `EndingBefore`
+            // constraint and not a `StartingAfter` constraint
+            if (!string.IsNullOrEmpty(options.EndingBefore) && string.IsNullOrEmpty(options.StartingAfter))
```

Comment:
What if you want to paginate between 2 ids? I never fully understand how they relate to each other.

Reply:
Our API raises if you pass both. 🤷‍♂